feat: terminology added (#184)
open-sauced/docs: docs/maintainers/setting-up-a-new-repository.md

- 👀 needs-triage (green background one)
- other potential duplicates if the above race condition is different

> In this context, `race condition` refers to a situation where multiple labels are being deleted simultaneously. This can cause issues if the order in which the labels are deleted affects the final outcome. Therefore, it's important to ensure that the deletion of labels is properly synchronized and controlled to avoid any race conditions.

In other words, if two of the directories (e.g., `👀 needs-triage` and `documentation` are deleted at the same, it is possible that the third directory(`potential duplicates`) will not be deleted.
To avoid the `race condition`, the code must delete the directories in a specific order. For example, it could delete the `documentation directory` first, then the `👀 needs-triage` directory, and then the other `potential duplicates` directory.

Here is an example of how to delete the directories in a specific order:
```bash
rm -rf documentation
rm -rf 👀-needs-triage
rm -rf other-potential-duplicates
```

> This code will ensure that all three directories are deleted, even if the code is interrupted while it is running.

Reviewer: I am not sure this is necessary. The mention of a race condition is a one-liner bullet point, and this explanation is longer than the original content trying to explain how to copy labels over.

Basically, at what point are we distracting from the original goal of this section? Perhaps all this extra would be a better fit for a blog post of glossary.

- 👀 needs-triage (green background one)
- other potential duplicates if the above race condition is different

> In this context, `race condition` refers to a situation where multiple labels are being deleted simultaneously. This can cause issues if the order in which the labels are deleted affects the final outcome. Therefore, it's important to ensure that the deletion of labels is properly synchronized and controlled to avoid any race conditions.

> In otherwords, the `race condition` is that the code is trying to delete three different directories: `documentation`, `👀 needs-triage`, and other `potential duplicates`. If two of the directories are deleted at the same time, it is possible that the third directory will not be deleted.
> To avoid this race condition, the code should delete the directories in a specific order. For example, it could delete the `documentation directory` first, then the `👀 needs-triage` directory, and then the other `potential duplicates` directory.

Reviewer 2: [suggestion] Consider shortening this paragraph a bit to show that it is simplifying the concept of `race condition`. 
```suggestion
In other words, if two of the directories (e.g., `👀 needs-triage` and `documentation` are deleted at the same, it is possible that the third directory(`potential duplicates`) will not be deleted.
To avoid `race condition`, the code must delete the directories in a specific order. For example, it could delete the `documentation directory` first, then the `👀 needs-triage` directory, and then the other `potential duplicates` directory.
```

Reviewer 3: In other words, if two of the directories (e.g., `👀 needs-triage` and `documentation` are deleted at the same time, it is possible that the third directory(`potential duplicates`) will not be deleted.
